### DISCUSSION TITLE
Graph plotting

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#281] [Q&A]
I am working on tensile_test examples, and not able to plot graph on paraview; it is showing nothing when I am trying to add a plot over line filter; I am attaching the screenshot below.

Please help me.
![sample](https://user-images.githubusercontent.com/110670051/202633832-2637ca01-1c39-4db0-b37d-9a242bfec875.png)

**Comment by ppandit95** [comment]
Hi Nikhil23121998,

So one of the question that might cause this issue could be that 

Have you applied the filter to convert cell data to point data as well as have you selected a specific node from selection toolbar?

Hope this helps

Regards
Pushkar

**Reply by Nikhil23121998** [reply]
I tried, but it is not working.

**Comment by jritta** [comment]
Hi Nikhil23121998,

The problem is that the data exported by Peridigm are not available as complete field on the geometry. In Peridigm a collocation method is used to discretize the linear momentum balance. Thus we only have values at the collocation points which are then directly exported. When your geometry is convex you can use the ``Delauney 2D`` or ``Delauney 3D`` filter to interpolate the data to the full geometry and then use the ``Plot over line`` filter.

Best,
Jonas

**Reply by ppandit95** [reply]
Thanks @jritta, it was helpful to know that for me as I didn't knew that aspect....
